cluster-autoscaler/go.mod

```diff
@@ -10,30 +10,32 @@ require (
 	github.com/Azure/go-autorest/autorest/date v0.3.0
```

Comment:
The go version is not updated by the `update-vendor.sh` script, even though it should be. I'll fix the script, but for this PR could you manually bump the go version in `go.mod` to 1.16, to keep it in line with k8s?

Reply:
Some of the changes didn't add properly, I have rerun.